### PR TITLE
Update S/4HANA Cloud Connectity Tutorial

### PR DESCRIPTION
**Pull request by mervey45**

### CODE REVIEW

File: tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md [hunk 1]
```diff
@@ -46,155 +38,135 @@ author_profile: https://github.com/mervey45
...
+   3. Check connection.
 
       ![Create destination with basic authentication](connection.png)
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 3: ](Create Custom CDS View in SAP S/4HANA Cloud)]
-  1. Logon to your S/4HANA Cloud system, navigate to Extensibility and open the Custom CDS Views tile.
+  1. Logon to your SAP S/4HANA Cloud system, navigate to Extensibility and open the Custom CDS Views tile.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md:67]: "log on" or "log in". Logon is a noun.

File: tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md [hunk 2]
```diff
@@ -21,23 +21,15 @@ author_profile: https://github.com/mervey45
 ---
 
 [ACCORDION-BEGIN [Step 1: ](Copying the inbound service URL)]
-  1. Logon to SAP S/4HANA Cloud system as administrator.
+  1. Logon to SAP S/4HANA Cloud system as administrator and select **Communication Arrangement** in the **Communication Management** section.
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md:24]: "log on" or "log in". Logon is a noun.

File: tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md [hunk 3]
```diff
@@ -46,155 +38,135 @@ author_profile: https://github.com/mervey45
...
+
+ 14. Navigate to **Elements**, select **Add** > **Elements**.
+
+       ![Create Custom CDS View in SAP S/4HANA Cloud](cds13.png)
+
+ 15. Search for `BusinessPartner`, select it and click **OK**.
+
+       ![Create Custom CDS View in SAP S/4HANA Cloud](cds14.png)
+
+ 16. Add also following elements to your CDS view:
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md:127]: Add also the following elements to your CDS view.

or 
Also, add the following elements to your CDS view.

Anyway, please add "the".

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md (hunk 1 above)
Comment (mervey45) [reply tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md]: I updated this.

File: tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md (hunk 2 above)
Comment (mervey45) [reply tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md]: I updated this.

File: tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md (hunk 3 above)
Comment (mervey45) [reply tutorials/abap-environment-s4hanacloud-service-consumption/abap-environment-s4hanacloud-service-consumption.md]: I updated this.

**Pull request merged**